arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Anytrust orbit chains keyset generation'
+sidebar_label: 'Keyset Generation for Anytrust Orbits'
+description: 'Keyset generation and setup for Anytrust Orbit chains'
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Anytrust Orbit chains.'
+sidebar_position: 1
+---
+**Anytrust** chains, relies on an external Data Availability Committee (hereafter, "the Committee") to store data and provide it on demand, instead of using the parent chain as Data Availability (DA) layer. 
+
+The Committee comprises N members, among whom it is presumed by AnyTrust that a minimum of two maintain integrity. Consequently, in scenarios where N - 1 members of the Committee pledge to grant access to specific data, it is guaranteed that at least one member among those committing to this promise upholds honesty.
```

Comment:
The minimum number of trusted committee members is configurable by the "assumed-honest" parameter in the keyset.